pyomo/contrib/pynumero/sparse/utils.py

```diff
@@ -0,0 +1,155 @@
+from pyomo.contrib.pynumero.sparse.base import SparseBase
+from pyomo.contrib.pynumero.sparse.extract import triu, tril
+from scipy.sparse.sputils import isscalarlike
+
+import numpy as np
+
+
+"""
+def read_matrix(filename, matrix_format='coo', verbose=False, offset=1):
```

Comment:
Is there a reason this is all commented out instead of being removed?

Reply:
Removed comment